bin/yosys

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+cd yosys2digitaljs-server && yarn start
```

Comment:
What is a use case of this file?

Reply:
Oh, actually to Create Verilog Circuits you need to run this express server "yosys2digitaljs-server"

Its just a handy shell script to run that express server. (For good DX.)

Just do `bin/yosys` everytime you want to start the server instead of doing `cd yosys2digitaljs-server && yarn start`